pkg/cidata/cidata.TEMPLATE.d/user-data

```diff
@@ -64,3 +64,13 @@ ca-certs:
     {{- end }}
   {{- end }}
 {{- end }}
+
+{{- if .BootCmds }}
+bootcmd:
+  {{- range $cmd := $.BootCmds }}
+  - |
+    {{- range $line := $cmd.Lines }}
+    {{ $line }}
+    {{- end }}
+  {{- end }}
+{{- end }}
```

Comment:
Ok, this is nit-picking, but the list element is double-indented, it looks like this:

```yaml
bootcmd:
  - |
    #!/bin/bash
    ...
```

when it should look like this:

```yaml
bootcmd:
- |
  #!/bin/bash
  ...
```

```suggestion
{{- if .BootCmds }}
bootcmd:
  {{- range $cmd := $.BootCmds }}
- |
    {{- range $line := $cmd.Lines }}
  {{ $line }}
    {{- end }}
  {{- end }}
{{- end }}
```